app/views/users/profile.html.erb

```diff
@@ -341,6 +341,25 @@
         profile.value.name = "<%= @user.name %>"
         profile.fetch_maps();
     })();
+
+  $('#follow_form').on('ajax:success', '#profile-follow-button', function(data, status, xhr)
+  {
+    notyNotification('relax', 3000, 'success', 'top', 'You started following <%=@user.username%>');
+    $('#follow_form').html('<a id="profile-unfollow-button" class="btn btn-default" data-method="delete" data-remote="true" href="/relationships/<%=@user.id%>">Unfollow</a>');
+  });
+  $('#follow_form').on('ajax:success', '#profile-unfollow-button', function(data, status, xhr)
+  {
+    notyNotification('relax', 3000, 'success', 'top', 'You unfollowed <%=@user.username%>');
+    $('#follow_form').html('<a id="profile-follow-button" class="btn btn-primary" data-method="post" data-remote="true" href="/relationships" data-params="followed_id=<%=@user.id%>">Follow</a>');
+  });
+  $('#follow_form').on('ajax:error', '#profile-follow-button', function(data, status, xhr)
+  {
+    notyNotification('relax', 3000, 'error', 'top', 'Error in following');
+  });
+  $('#follow_form').on('ajax:error', '#profile-unfollow-button', function(data, status, xhr)
+  {
+    notyNotification('relax', 3000, 'error', 'top', 'Error in following');
```

Comment:
this should be unfollowing? right? Also, change this message to follow the pattern if you make this change